packages/liveblocks-react-blocknote/src/version-history/HistoryVersionPreview.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+import type { BlockNoteEditor } from "@blocknote/core";
+import { HistoryVersionPreview as TipTapHistoryVersionPreview } from "@liveblocks/react-tiptap";
+
+type HistoryVersionPreviewProps = Omit<
+  Parameters<typeof TipTapHistoryVersionPreview>[0],
+  "editor"
+> & {
+  editor: BlockNoteEditor<any, any, any>;
```

Comment:
These components only exist for fixing this type right? I'm wondering if there's a way we can fix the types without having these wrapper components. Maybe adding a declare in index.ts or something? Maybe this is less risky than that idea.

Reply:
It's not just the type, because the tiptap component requires `editor. _tiptapEditor`. 

Because this is an undocumented BlockNote API (to access the underlying tiptap editor), I don't want to ask consumers to access / pass this around.

By building our own "BlockNote" components (even if they just wrap the tiptap ones), we're also more future proof in case we want to make changes to the BlockNote, but not the TipTap implementation